src/Forum/client.ts

```diff
@@ -29,3 +35,14 @@ export async function createPost(post: ForumPost): Promise<string> {
   const { topic_id, topic_slug } = result
   return `${FORUM_URL}/t/${topic_slug}/${topic_id}`
 }
+
+function sanitizeTitle(title: string) {
+  return removeEmojis(title)
+}
+
+export function removeEmojis(text: string) {
+  return text.replace(
+    /([\u2700-\u27BF]|[\uE000-\uF8FF]|\uD83C[\uDC00-\uDFFF]|\uD83D[\uDC00-\uDFFF]|[\u2011-\u26FF]|\uD83E[\uDD10-\uDDFF])/g,
+    ' '
```

Comment:
Should we leave a blank space where there are emojis or just remove them?

Reply:
Both for me are the same, added the whitespace to signal that something was actually there.

Reply:
Removed whitespace